sdk/metric/selector/simple/simple_test.go

```diff
@@ -41,30 +45,34 @@ func oneAgg(sel export.AggregatorSelector, desc *metric.Descriptor) export.Aggre
 	return agg
 }
 
+func testFixedSelectors(t *testing.T, sel export.AggregatorSelector) {
+	require.NotPanics(t, func() { _ = oneAgg(sel, &testValueObserverDesc).(*lastvalue.Aggregator) })
+	require.NotPanics(t, func() { _ = oneAgg(sel, &testCounterDesc).(*sum.Aggregator) })
+	require.NotPanics(t, func() { _ = oneAgg(sel, &testUpDownCounterDesc).(*sum.Aggregator) })
+	require.NotPanics(t, func() { _ = oneAgg(sel, &testSumObserverDesc).(*sum.Aggregator) })
+	require.NotPanics(t, func() { _ = oneAgg(sel, &testUpDownSumObserverDesc).(*sum.Aggregator) })
+}
```

Comment:
Rather than `require.NotPanics()`, have we considered using `require.IsType()`?

```go
        require.IsType(t, (*lastvalue.Aggregator)(nil), oneAgg(ex, &testValueObserverDesc))
        require.IsType(t, (*sum.Aggregator)(nil), oneAgg(ex, &testCounterDesc))
        require.IsType(t, (*sum.Aggregator)(nil), oneAgg(ex, &testUpDownCounterDesc))
        require.IsType(t, (*sum.Aggregator)(nil), oneAgg(ex, &testSumObserverDesc))
        require.IsType(t, (*sum.Aggregator)(nil), oneAgg(ex, &testUpDownSumObserverDesc))
```

It gives a nicer error message when something is not of the correct type.

Reply:
Done.